runconfig/config.go

```diff
@@ -20,6 +20,7 @@ type Config struct {
 	Hostname        string                // Hostname
 	Domainname      string                // Domainname
 	User            string                // User that will run the command(s) inside the container
+	ArgsEscaped     bool                  `json:",omitempty"` // True if command is already escaped
```

Comment:
Where is the best place to put it such that it can be sent from dockerfile/dispatchers.go to the execdriver where it needs to be processed? I put it in Config, as that seemed to be the location for similar parameters. And yes, it is platform specific.